surveys/2021-annual-survey/questions.md

```diff
@@ -219,70 +219,213 @@ TODO ask about blockers for participation?
 
 ## Rust at work
 
-TODO see previous surveys
+> This section is largely in service of the [contexts](./design/contexts.md) design document.
 
-Are you using Rust at work
+### Are you using Rust at work
 
-- yes, >90% of my coding
-- yes, it's one of a number of languages I use and I use it regularly
-- yes, but I only use it occasionally
-- no, but it is likely in the future
-- no
+Select one:
+- Yes, for the majority of my coding
+- Yes, it's one of a number of languages I use and I use it regularly
+- Yes, but I only use it occasionally
+- No, but it is likely in the next year
+- No, I use other programming languages at work
+- Not applicable (i.e., I don't write software professionally, I am student, etc.)
+
+> **justification**
+>
+> We want to establish what percentage of those who could possibly use Rust in a professional setting
+> are using Rust in a professional setting. This is most interesting over time. 
+> Answers to this question should be combined with whether the respondent has ever used Rust.
 
+**TODO**: explain why the following questions are not asked
 Is Rust used in the company where you work?
 
-- yes by > 100 devs
-- yes < 100
-- yes < 50
-- yes < 20
-- yes < 10
-- yes < 5
+### To why extent is Rust currently being used by your company?
+
+Select one:
+- My company uses Rust for a large portion of production projects.
+- My company uses Rust for a small portion of production projects.
+- My company uses Rust only for non-production projects (e.g., tooling).
+- My company has actively experimented with Rust for use in production projects.
+- My company has actively experimented with Rust for use in non-production projects.
+- My company has seriously discussed but not experimented with using Rust.
+- My company has not seriously considered Rust for any use.
+- I am unsure whether my company has considered using or currently uses Rust.
+- I don't work for a company or my company does not develop software of any kind.
+
+> **justification**
+>
+> We want to establish how reliant companies are on Rust.
+
+### In what technology domains is Rust used at your company?
+
+Select as many as apply:
+- backend application 
```

Comment:
I would rephrase this, 'backend' is heavily overloaded and (to me at least) overlaps significantly with web application